src/openMVG_Samples/features_repeatability/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ target_link_libraries(openMVG_sample_main_repeatability_dataset
   openMVG_system
   openMVG_multiview
   vlsift
-  ${POPSIFT_LIBRARIES}
+  PopSift::popsift # ${POPSIFT_LIBRARIES}
```

Comment:
Actually, this would not be necessary: popsift is not used "directly" here (ie there is no include <popsift..>), so *in theory* it should be added directly by the `openMVG_features` if the dependencies are declared correclty